content/reference-rippled.md

```diff
@@ -10955,6 +10957,182 @@ The response follows the [standard format](#response-formatting), with a success
 
 * Any of the [universal error types](#universal-errors).
 
+
+
+
+# Peer Protocol #
+
+Servers in the Ripple Consensus Ledger communicate to each other using the Ripple peer protocol, also known as RTXP. Peer servers connect via HTTPS and then perform an [HTTP upgrade](https://tools.ietf.org/html/rfc7230#section-6.7) to RTXP. (For more information, see the [Overlay Network article in the `rippled` repository](https://github.com/ripple/rippled/blob/906ef761bab95f80b0a7e0cab3b4c594b226cf57/src/ripple/overlay/README.md#handshake).)
+
+## Configuring the Peer Protocol ##
+
+In order to participate in the Ripple Consensus Ledger, `rippled` servers connects to arbitrary peers using the peer protocol. (All such peers are treated as untrusted, unless they are [clustered](tutorial-rippled-setup.html#clustering) with the current server.)
+
+Ideally, the server should be able to send _and_ receive connections on the peer port. You should forward the port used for the peer protocol through your firewall to the `rippled` server. The [default `rippled` configuration file](https://github.com/ripple/rippled/blob/develop/doc/rippled-example.cfg) listens for incoming peer protocol connections on port 51235 on all network interfaces. You can change the port used by editing the appropriate stanza in your `rippled.cfg` file.
+
+Example:
+
+```
+[port_peer]
+port = 51235
+ip = 0.0.0.0
+protocol = peer
+```
+
+## Peer Crawler ##
+
+The Peer Crawler reports information about other servers a `rippled` server is connected to as peers. The [`peers` command](#peers) in the [WebSocket and JSON-RPC APIs](#websocket-and-json-rpc-apis) also returns a similar, more comprehensive set of information, but requires admin access to the server. The Peer Crawler response is available to other servers on a non-privileged basis through the Peer Protocol (RTXP) port.
```

Comment:
The sentence is like LOTR: it seems to end, but then keeps on going. The especially confusing part for me is the "peers" at the end, which refers back to "servers." But, like I said, there might not really be a way to improve the sentence (to my level of understanding, anyway). That said, I'm always in favor of explaining these nuances more verbosely upfront (for example, maybe with the explanation in your comment and a link to the "Concealing Peer Information" section where that exception is fully explained. My concern is that if I had to read the sentence more than once, other users probably will, too, and that could make a reader's experience just a little less than pleasant.